ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
<p>Is there a reason not to use <code>radians</code> and <code>degrees</code> from numpy?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
`radians` is the same as `deg2rad`, but `deg2rad` is a more descriptive name and in the [code](https://stackoverflow.com/questions/51801646/difference-between-deg2rad-and-radians-in-numpy) `radians` uses `deg2rad`. I wouldn't be surprised if numpy deprecates `radians`/`degrees` in the future since it is duplicated functions.

Reply:
Sure, I was more curious about the use of math vs either set of numpy functions (see above).

I'll close this specific comment though in favor of the general discussion above.